src/tools/dart-create.md

```diff
@@ -0,0 +1,113 @@
+---
+title: dart create
+description: Command-line tool for creating Dart projects.
+toc: false
+---
+
+The `dart create` command allows creating Dart projects
+from varying available templates through a command line
+or many IDE project creation workflows.
+
+{% include tools/dart-tool-note.md %}
+
+The command creates a directory with the specified name, 
+creates necessary files, configures static analysis, 
+and retrieves any necessary dependencies.
```

Comment:
I'd like the first screenful to show or hint at the commands that people are most likely to use — more like https://dart.dev/tools/dart-analyze. Maybe something like this:

-------------

When you run `dart create`, it first creates a directory with the project files. Then it gets package dependencies (unless you specify the `--no-pub` flag.) Here's an example of using `dart create` to create a directory named `my_cli` that contains a simple console app (the default template):

```terminal
$ dart create my_cli
```


To use a different template, add a template argument:

```terminal
$ dart create -t web-simple my_web_app
```

The following table shows the templates you can use:
...